### PR TITLE
add default entries in MANIFEST.MF file and extra Build-Time entry as well #2185

### PR DESCRIPTION
**Pull request by jchobantonov**

### CODE REVIEW

**Comment by jchobantonov** [comment]
@mrniko

File: redisson/src/main/java/org/redisson/cache/CacheValue.java [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/**
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/cache/CacheValue.java]: I fixed that. Please rebase

File: pom.xml [hunk 2]
```diff
@@ -247,24 +263,24 @@
...
-         	<version>3.0.1</version>
-         	<executions>
-                   <execution>
-                     <id>attach-javadocs</id>
-                     <goals>
-                         <goal>jar</goal>
-                     </goals>
-                   </execution>
-         	</executions>
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (mrniko) [inline pom.xml]: Please remain only changed lines

File: pom.xml [hunk 3]
```diff
@@ -206,16 +222,16 @@
...
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-release-plugin</artifactId>
-            <version>2.5.3</version>
-   		<configuration>
-   			<autoVersionSubmodules>true</autoVersionSubmodules>
-   			<useReleaseProfile>true</useReleaseProfile>
-   			<releaseProfiles>release</releaseProfiles>
-   			<goals>deploy</goals>
-   		</configuration>
+            <plugin>
```
Comment (mrniko) [inline pom.xml]: Please remain only changed lines

File: pom.xml [hunk 4]
```diff
@@ -139,27 +139,43 @@
...
-       		<artifactId>nexus-staging-maven-plugin</artifactId>
-       		<version>1.6.8</version>
-       		<extensions>true</extensions>
-       		<configuration>
-       			<serverId>ossrepo</serverId>
-       			<nexusUrl>https://oss.sonatype.org/</nexusUrl>
-       			<autoReleaseAfterClose>true</autoReleaseAfterClose>
-       		</configuration>
-       	</plugin>
+            <plugin>
```
Comment (mrniko) [inline pom.xml]: Please remain only changed lines

File: pom.xml [hunk 5]
```diff
@@ -75,26 +75,26 @@
...
-                                <manifestEntries>
-                                  <Implementation-Build>${implementation.build}</Implementation-Build>
-                                  <Implementation-Build-Date>${maven.build.timestamp}</Implementation-Build-Date>
-                                  <X-Compile-Source-JDK>${javac.src.version}</X-Compile-Source-JDK>
-                                  <X-Compile-Target-JDK>${javac.target.version}</X-Compile-Target-JDK>
-                                </manifestEntries>
-                              </archive>
-                            </configuration>
-                          </execution>
+                            <execution>
```
Comment (mrniko) [inline pom.xml]: Please remain only changed lines

**Comment by jchobantonov** [comment]
closing PR in favor of https://github.com/redisson/redisson/pull/2190

**Pull request closed without merging**